docs/markdown/Rewriter.md

```diff
@@ -77,6 +77,17 @@ and print an error message.
 For more information see the help output of the rewriter target
 command.
 
+### Adding and removing `extra_files`
+
+*Since 0.61.0*
```

Comment:
Not sure about the version I should specify here, if I should specify at all.

Reply:
Always use the next Meson release version, so this is correct. If a PR doesn't make it the version must be updated.